--- OpenGTL/GTLCore/CMakeLists.txt.orig	2013-09-01 15:15:40.473049010 +0100
+++ OpenGTL/GTLCore/CMakeLists.txt	2013-09-01 15:15:54.383048198 +0100
@@ -110,7 +110,7 @@
 set(EXTRA_GTLCORE_LINK_OPTIONS -Wl,-u,__ZTVN4llvm3JITE )
 endif()
 
-target_link_libraries(GTLCore ${LLVM_LDFLAGS} ${LLVM_LIBS} ${EXTRA_GTLCORE_LINK_OPTIONS})
+target_link_libraries(GTLCore ${LLVM_LDFLAGS} ${LLVM_LIBS} ${EXTRA_GTLCORE_LINK_OPTIONS} dl pthread)
 
 # Set the ABI version of the library
 set_target_properties(GTLCore PROPERTIES VERSION ${OPENGTL_LIB_VERSION} SOVERSION ${OPENGTL_LIB_SOVERSION} )
