--- OpenGTL-0.9.16/OpenGTL/GTLCore/CMakeLists.txt.bero	2012-04-18 12:34:32.721801852 +0200
+++ OpenGTL-0.9.16/OpenGTL/GTLCore/CMakeLists.txt	2012-04-18 12:34:58.020802380 +0200
@@ -31,6 +31,7 @@ if(MSVC)
       ${LLVM_LIBRARIES_PATH}/LLVMSystem.lib )
 else(MSVC)
   FIND_LLVM_LIBS( ${LLVM_CONFIG_EXECUTABLE} "native bitwriter linker bitreader jit interpreter support ipo" LLVM_LIBS LLVM_NATIVE_OBJECTS )
+  STRING(REGEX REPLACE " " ";" LLVM_LIBS ${LLVM_LIBS})
 endif(MSVC)
 
 include_directories( ${LLVM_INCLUDE_DIR} )
